pkg/runs/runsDownload_test.go

```diff
@@ -429,6 +429,32 @@ func TestRunsDownloadWritesSingleArtifactToFileSystem(t *testing.T) {
 	assert.True(t, downloadedArtifactExists)
 }
 
+func TestRunsDownloadWritesSingleArtifactToFileSystemUsingTargetFolderPath(t *testing.T) {
+	// Given ...
+	runName := "U27"
+	runId := "xxx543xxx"
```

Comment:
can we change this to use RUN_27V2 in line 439 and on line 435 use id= "xxx987xxx" 
as RUN_U27 doesn't have a result so the folder name will have the timestamp in it so the assert will fail.
..or change the flder name in the assert but it would be easier to just change the run we pass in